library-no-op/src/main/java/com/chuckerteam/chucker/api/Chucker.kt

```diff
@@ -0,0 +1,26 @@
+package com.chuckerteam.chucker.api
+
+import android.content.Context
+import android.content.Intent
+
+/**
+ * No-op implementation.
+ */
+object Chucker {
+
+    val SCREEN_HTTP = 1
+    val SCREEN_ERROR = 2
+
+    val isOp: Boolean get() = false
```

Comment:
Instead of defining the Getter, should we directly set the value ?
`val isOp: Boolean = false`